thinc/layers/softmax.py

```diff
@@ -17,29 +17,40 @@ def Softmax(
     nI: Optional[int] = None,
     *,
     init_W: Callable = zero_init,
-    init_b: Callable = zero_init
+    init_b: Callable = zero_init,
+    normalize: bool = True,
```

Comment:
I would consider renaming this argument, too. Because in other layers where the constructor takes a `normalize` argument, it defines whether or not a `LayerNorm` should be chained to the layer. Could we call this something like `normalize_outputs`?

Reply:
Fixed.